specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-04-01/examples/ListSkus.json

```
@@ -0,0 +1,79 @@
+{
+  "parameters": {
+    "api-version": "2020-03-01",
```

Seems this should be `2020-04-01`? Please double check all other examples for similar issues
```suggestion
    "api-version": "2020-04-01",
```